client/cypress/integration/visualizations/pivot_spec.js

```diff
@@ -103,6 +103,7 @@ describe("Pivot", () => {
       cy.getByTestId("ExecuteButton").click();
 
       // assert number of rows is 12
+      cy.wait(250);
       cy.getByTestId("PivotTableVisualization").contains(".pvtGrandTotal", "12");
```

Comment:
Fine to leave it to fix if the test is broken, but usually it's not a good practice to do arbitrary `cy.wait`, because it adds up time to the tests and there's no guarantee that the element will be ready after that. Instead, the idea is to assert for something else that we know would be a "ready signal" in the UI itself (e.g.: notification, some message that change)

Do you happen to have a link, screenshot or video to the failure it had?

Reply:
Btw, it's possible you'll have to ignore the ESLint rule for the line if you want to keep the `cy.wait`

Reply:
That `cy.wait()` call was just my suggestion in Discourse, as I've seen it help (in another project) with specific tests if they commonly fail at a certain point.

So, this was just @gaecoli trying it out to see if it helps. :smile:

Reply:
> @justinclift restyled is not required ...

Oh cool, thanks. :smile:

Reply:
This is a timeout test. The default time of cypress cannot achieve the purpose, so we specify the timeout at the end when we click, so that the test can pass. I have checked the official documentation of cypress, and there is indeed such a situation.